platform/broadcom/platform-modules-inspur.mk

```diff
@@ -0,0 +1,16 @@
+# Inspur Platform modules
+
+INSPUR_SC5630EL_PLATFORM_MODULE_VERSION = 1.0
+
+
+export INSPUR_SC5630EL_PLATFORM_MODULE_VERSION
+
+INSPUR_SC5630EL_PLATFORM_MODULE = sonic-platform-inspur-sc5630el_$(INSPUR_SC5630EL_PLATFORM_MODULE_VERSION)_amd64.deb
+$(INSPUR_SC5630EL_PLATFORM_MODULE)_PLATFORM = x86_64-inspur_sc5630el-r0
+$(INSPUR_SC5630EL_PLATFORM_MODULE)_SRC_PATH = $(PLATFORM_PATH)/sonic-platform-modules-inspur
+$(INSPUR_SC5630EL_PLATFORM_MODULE)_DEPENDS += $(LINUX_HEADERS) $(LINUX_HEADERS_COMMON)
+SONIC_DPKG_DEBS += $(INSPUR_SC5630EL_PLATFORM_MODULE)
+
+SONIC_STRETCH_DEBS += $(INSPUR_SC5630EL_PLATFORM_MODULE)
```

Comment:
remove this one? now we are using buster.

Reply:
OK, done